src/Controller/EnMarche/NewsletterController.php

```diff
@@ -114,4 +115,17 @@ public function invitationSentAction(Request $request)
             'invitations_count' => $invitationsCount,
         ]);
     }
+
+    public function renderNewsletterFormAction(array $options): Response
+    {
+        return $this->render('newsletter/form.html.twig', array_merge([
```

Comment:
CS: This is not readable.

You can create intermediary variables as a solution.